src/ProjectOrigin.PedersenCommitment/Commitment.cs

```diff
@@ -0,0 +1,45 @@
+using System.Numerics;
+using System.Runtime.Serialization;
+
+namespace ProjectOrigin.PedersenCommitment;
+
```

Comment:
A commented line referring to the project origin registry documentation on [Pedersen Commitment](https://github.com/project-origin/registry/blob/main/doc/pedersen_commitments/pedersen-commitments.md)